base/abstractarraymath.jl

```diff
@@ -449,9 +449,10 @@ See also [`eachrow`](@ref), [`eachcol`](@ref), and [`selectdim`](@ref).
      This function requires at least Julia 1.1.
 """
 @inline function eachslice(A::AbstractArray; dims)
-    length(dims) == 1 || throw(ArgumentError("only single dimensions are supported"))
+    length(dims) == 1 || throw(ArgumentError("only single dimensions are supported in eachslice"))
```

Comment:
```suggestion
    length(dims) == 1 || throw(ArgumentError("only single dimensions are supported"))
```
I don't think adding that message helps. The stacktrace will tell you